src/compat.rs

```diff
@@ -50,17 +50,17 @@ pub(crate) struct AllowStd<S> {
 //
 // Don't ever use this from multiple tasks at the same time!
 pub(crate) trait SetWaker {
```

Comment:
There are some warnings (in certain configurations) that this trait is never used

Reply:
Also `use SinkExt` in `examples/server-headers.rs` and the `closing` bool in `src/lib.rs` inside `WebSocketStream`